bundles/org.openhab.core.audio/src/main/java/org/openhab/core/audio/FileAudioStream.java

```diff
@@ -125,4 +133,11 @@ public synchronized void reset() throws IOException {
     public InputStream getClonedStream() throws AudioException {
         return getInputStream(file);
     }
+
+    @Override
+    public void dispose() throws IOException {
```

Comment:
Note that it will require a change the pico and mac TTS services to build a FileAudioStream rather than a FixedLengthAudioStream to have the temporary file deleted.

Reply:
Or to implement Closable in the existing FixedLengthAudioStream.

Reply:
> Or simply to implement Closable in the existing FixedLengthAudioStream.

Yes, this one seems easier.